x-pack/plugins/cases/public/methods/get_cases_context.tsx

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
lazy component to load the CasesContext.

The `CasesContext` is required to render the cases flyout and eventually the modals. You use the useCreateCaseFlyout hook to interact with this context.

Plugins should wrap the components that will show the flyout or modal on this.